application/models/adjacency_list_model.php

```diff
@@ -38,6 +38,7 @@ public function __construct()
 		$this->load->database();
 		$this->load->config('adjacency_list', TRUE);
 		$this->lang->load('adjacency_list');
+		$this->load->helper('security');
```

Comment:
It's not a real place for this helper - I would rather encourage people to load it automatically. For now - please delete this line. 
Afterall... we should stop using `xss_clean` on input.